pkg/agent/route/route_windows.go

```diff
@@ -36,6 +36,18 @@ const (
 	outboundFirewallRuleName = "Antrea: accept packets to local Pods"
 )
 
+var (
+	// ServiceGWVirtuallP is a virtual IP for redirecting ClusterIP Service traffic from host network to OVS.
+	// The traffic will be routed to antrea-gw0 via the virtual IP. It is added to use a single ARP responder
```

Comment:
Wait a min - I only saw you added a neighbor entry, but an ARP responder flow? If we should revise this comment and other descriptions?

Reply:
Updated to net neighbor.